Build/linq2db.LINQPad.nuspec

```diff
@@ -12,7 +12,7 @@
 		<dependencies>
 			<group targetFramework=".NETCoreApp3.1">
 				<dependency id="LINQPad.Reference"                         version="1.3.0"     />
-				<dependency id="linq2db"                                   version="4.0.1"     />
+				<dependency id="linq2db"                                   version="4.1.0"     />
```

Comment:
npgsql was also updated

Reply:
and System.Text.Json

Reply:
Thanks! One of these days I'll remember my head